scripts/afterfx/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "cursor-recorder-for-afterfx",
+  "version": "0.0.1",
+  "description": "",
+  "main": "script.ts",
+  "scripts": {
+    "watch": "tsc -w",
+    "build": "tsc && cd .. && sed -i '/\\$/d' cursor_recorder_for_afterfx.jsx"
+  },
+  "author": "Jakub Koralewski",
+  "license": "MIT",
+  "dependencies": {
+    "types-for-adobe": "^1.5.0"
```

Comment:
Package.json not needed

Reply:
it is needed noob, the dependency is not needed

Reply:
resolved https://github.com/JakubKoralewski/cursor-recorder/pull/5/commits/c584961f14cbc3f7f5dc751f6d96b94d149f5586